README.md

```diff
@@ -93,20 +93,39 @@ Multi-noding (Sharing CPU, RAM, Uplink or storage among nodes) is not allowed.**
 
    You can use the text editor of your choice (e.g. `nano` or `vim` on Linux)
 
-4. Download the docker-compose file
+4. Download the `docker-compose.yml` file:
 
    ```bash
    curl -s https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/docker-compose.yml -o docker-compose.yml
    ```
 
-5. Launch it:
+5. Download the `docker_compose_update.sh` script and make it executable:
+
+   ```bash
+   curl -s https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/docker_compose_update.sh -o docker_compose_update.sh
+   chmod +x docker_compose_update.sh
+   ```
+
+6. Set up the cron job to auto-update the `docker-compose.yml` file:
```

Comment:
Can we let them copy/paste a script instead of going through the interactive crontab editor?

https://stackoverflow.com/a/9625233/2498782